neptune/new/internal/backends/hosted_neptune_backend.py

```diff
@@ -165,10 +170,14 @@ def __init__(self, credentials: Credentials, proxies: Optional[Dict[str, str]] =
             build_operation_url(self._client_config.api_url, self.LEADERBOARD_SWAGGER_PATH),
             self._http_client
         )
-        self.artifacts_client = create_swagger_client(
-            build_operation_url(self._client_config.api_url, self.ARTIFACTS_SWAGGER_PATH),
-            self._http_client
-        )
+        try:
+            self.artifacts_client = create_swagger_client(
+                build_operation_url(self._client_config.api_url, self.ARTIFACTS_SWAGGER_PATH),
+                self._http_client
+            )
+        except Exception:
```

Comment:
A lot of different things can go wrong here. Exception does not necessarily mean artifacts are not supported.
Maybe we should add list of supported features to config returned from backend.
@Herudaio how urgent is it to make new clients work for onprems?